.github/workflows/ci.yml

```diff
@@ -207,7 +207,7 @@ jobs:
         'project-lint-jobs',
         'project-test-jobs',
       ]
-    if: ${{ always() && github.event_name == 'pull_request' }}
+    if: ${{ !cancelled() && github.event_name == 'pull_request' }}
```

Comment:
Ensures `Evaluate Project Job Statuses` will not start if workflow cancellation is requested.